pubsub-protocol-binding.md

```diff
@@ -113,8 +113,8 @@ The receiver of the event can distinguish between the two content modes by
 inspecting the [Pub/Sub Attribute `Content-Type`][content-type] of the
 [Pub/Sub message][pubsub-message]. If the attribute is present and its value is prefixed 
 with the CloudEvents media type `application/cloudevents`, indicating the use of a known
-[event format](#14-event-formats), the receiver uses _structured_ mode,
-otherwise it defaults to _binary_ mode.
+[event format](#14-event-formats), the receiver is expected to use _structured_ mode,
+otherwise it SHOULD default to _binary_ mode.
```

Comment:
```suggestion
otherwise it MUST default to _binary_ mode.
```

Reply:
My reading of the spec implies this is a MUST, but feel free to make this SHOULD if you disagree.

Reply:
is a MUST you are right...

Reply:
done